src/generic/datavgen.cpp

```diff
@@ -825,6 +825,7 @@ class wxDataViewMainWindow: public wxWindow
 
     int GetCountPerPage() const;
     int GetEndOfLastCol() const;
+    int GetColumnStart(int column) const;
```

Comment:
It would be nice to add a comment here saying that this function can be called with `column == -1` and just returns `-1` in this case.

Alternatively, maybe it would be more clear if it could be only called with a valid column (and then always returned a valid value) and then not call it below when column is not specified.